eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -52,46 +52,16 @@ parameters:
       SubscriptionConfiguration: $(sub-config-gov-test-resources)
     China:
       SubscriptionConfiguration: $(sub-config-cn-test-resources)
-- name: Platforms
+- name: MatrixConfigs
   type: object
```

Comment:
As a side question do we expect we will use this matrix implementation for the nightly and ci test matrix at https://github.com/Azure/azure-sdk-for-net/blob/master/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L158 or do we think that is simple enough to stick with standard yml matrix?

Reply:
I think it makes sense to migrate ci to the matrix config as well.